### PR TITLE
Feature/wal 650

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/router/AppRouter.tsx [hunk 1]
```diff
@@ -4,22 +4,31 @@ import SSICredentialIssueRequestPage from '../pages/SSICredentialIssueRequestPag
...
-                <Route path="/" element={<SSILandingPage/>}/>
+                {/*<Route path="/" element={<SSILandingPage/>}/>*/}
+                <Route path="/" element={<SSICredentialsLandingPage/>}/>
                 <Route path={"/information/request"} element={<SSIInformationRequestPage/>}/>
                 <Route path={"/information/success"} element={<SSIInformationSuccessPage/>}/>
                 <Route path={"/credentials/verify/request"} element={<SSICredentialVerifyRequestPage/>}/>
                 <Route path={"/credentials/issue/request"} element={<SSICredentialIssueRequestPage/>}/>
                 <Route path={"/credentials/issue/success"} element={<SSICredentialIssuedSuccessPage/>}/>
                 <Route path={"/download"} element={<SSIDownloadPage/>}/>
+                {config.credentials.map(value => (
```
Comment (nklomp) [inline packages/oid4vci-demo-frontend/src/router/AppRouter.tsx]: What is this? The minicard overview is a single page listing multiple credentials with static links to other environments. This looks like dynamic routes based on credential names for a single environment

**Review by nklomp — COMMENTED** [review]
Just a remark on the routes/router.

How would the page be displayed and what's up with the different routes for the credntials?

File: packages/oid4vci-demo-frontend/src/router/AppRouter.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/router/AppRouter.tsx]: i have no clue what this is

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/router/AppRouter.tsx]: seems to be working without, so removed

File: packages/oid4vci-demo-frontend/src/configs/sphereon.json [hunk 2]
```diff
@@ -131,6 +131,63 @@
         }
       }
     },
+    "SSICredentialsLandingPage": {
```
Comment (nklomp) [inline packages/oid4vci-demo-frontend/src/configs/sphereon.json]: Shouldn't this be in a file called knb.json? It has little to do with the generic Sphereon demo.

I am okay with leaving it for now if it doesn't interfere with the generic Sphereon demo, but what is the path/url for this page then?

**Review by nklomp — APPROVED** [review]
Some remark/question about the Sphereon config

File: packages/oid4vci-demo-frontend/src/configs/sphereon.json (hunk 2 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/sphereon.json]: it's now in a file called knb.json as indeed that makes more sense. and as discussed, there is a /landing for the credential landing page

**Review by nklomp — APPROVED** [review]

**Pull request merged**